content/docs/source-git/_index.md

```diff
@@ -39,12 +40,13 @@ such a thing:
 
 ### Spec file
 
-I'm going to answer the elephant in the room right away: "Do we really need to
-have a spec file in our upstream repo or a source-git repo?!" — No, you don't,
-though storing the spec file directly in the repository is the most convenient.
-Alternatively, packit needs to have a way to obtain the spec file, so, you can:
+We are going to answer the elephant in the room right away: "Do we really need to
+[have a spec file in our upstream repo or a source-git repo?!]({{< ref "faq#how-can-i-download-rpm-spec-file-if-it-is-not-part-of-upstream-repository" >}})"
+No, you don't, though storing the spec file directly in the repository
+will save you a lot of trouble. Alternatively, packit needs to have a way to
```

Comment:
Shouldn't we capitalise `Packit`? Written like this it can be confusing for readers who are already not familiar with it.

Reply:
no strong preference from my side, I just capitalized all

Reply:
:rocket: